kate/grid/src/grid.rs

```diff
@@ -30,11 +30,11 @@ pub struct ColumnMajor<A> {
 
 impl<A> Grid<A> for RowMajor<A> {
 	fn width(&self) -> usize {
-		self.dims.width()
+		self.dims.cols().get().into()
```

Comment:
@fmiguelgarcia any reason for this change? I specifically switched to using width/height as Aleksandar/Kailas/others and I find the cols/rows terminology somewhat confusing mind-bending to work with.

Reply:
Added both methods `.weight() -> usize` and `.height() -> usize`.
As we've unified `Dimension` from grid and recovery both access members are used. 
att @Aphoh